dss_client/scripts/build.sh

```diff
@@ -48,11 +49,21 @@ rm -f "$DSS_CLIENT_DIR"/*.tgz
 #. "$SCRIPT_DIR/load_gcc.sh"
 source /opt/rh/devtoolset-11/enable
```

Comment:
Why this path is hard coded, parameterize it

Reply:
this is the build script calling cmake/make. We cannot parameterize every where. This is the environment settings